### DISCUSSION TITLE
multiple tc420 on one raspberry

### DISCUSSION THREAD

**Discussion by matrix1000** [wallneradam/tc420#5] [Q&A]
Hi,

many thanks for sharing your code. 
It works perfect for me on my raspberry pi with one tc420.
But if I try to connect a second, one or both tc420 are freezing.

Have you any idea, how to connect multiple tc420 on one raspberry?

**Comment by wallneradam** [comment]
Hi, at the moment it finds the 1st TC420 by vendor and device id and using that. Though it should not be hard to make it work with 2 or more devices as well. I'm going to make a test version with this functionality. Could you help me to test it? I have only one device.

**Comment by matrix1000** [comment]
Hi, thanks for your answer.
Sure, I will help you if I can.
The main problem is, that the tc420 crashes, if I put two in the raspberry.

**Comment by moeff** [comment]
I will test it too, i use it curruntly only for timesync and have also 2 tc420

**Reply by matrix1000** [reply]
Do you have the same problems with crashing tc420?
